include/odp/api/spec/queue_types.h

```diff
@@ -130,14 +131,14 @@ typedef struct odp_queue_capability_t {
 	  * types are used simultaneously. */
 	uint32_t max_queues;
 
-	/** Maximum number of ordered locks per queue */
-	uint32_t max_ordered_locks;
+	/** @deprecated  Use odp_schedule_capbility_t.max_ordered_locks */
```

Comment:
typo: _capability_t

Also "_capability_t.foo" is not very readable. I'd prefer:

"Use 'foo' field of odp_schedule_capability_t instead"

Reply:
ack